features.md

```diff
@@ -47,6 +47,13 @@ As of current version concurrency is not yet implemented.
 
 ## Decrypt
 
+Mechanisms:
+
+- AES-CBC
+- RSA-X-509
```

Comment:
What is "RSA-X-509"? I assume you mean X.509 but still not sure which mechanism this is.

Reply:
It corresponds to the RSA_RAW on the NetHSM